protocol/ServerPackets.hpp

```diff
@@ -138,6 +139,19 @@ namespace protocol
     };
     std::shared_ptr<ChangeDifficulty> parseChangeDifficulty(std::vector<uint8_t> &buffer);
 
+    struct ChatMessage : BaseServerPacket
+    {
+        std::string message;
+        // I think this is a string, it is marked as an 'Instant' in the protocol
```

Comment:
I think this is a long like in the "chat command" packet

Reply:
https://docs.oracle.com/javase/8/docs/api/serialized-form.html#java.time.Instant
Ok, we are both wrong x)
I will do that tomorrow